domains/i.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "kit2d2alt",
+           "email": "",
+           "discord": "1185040459156959251",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.p-GS6VFfsqZ3Efgec-u7irlX7K27bZtYExSWtJYopjv4ebMA8wgLdCif3ZQzU6i6PyHO4Uo-vzaC5AQIhr3v93B_o_QFhwrLPv305g-tsouE0TEzAt4gYrGY_zG4bXR4UnDROlnxsfBogijpApQdysTMsG7UZidLiiCeae097qgNl41a3ppGCKAuhqIXz1xGsbkx3HfPyAKGhWa-7kALJHUkBbI8ZCmYaC_TfVmIIATkHUj7R_VETZ03ijtBupW00LwAs5i6E4myNm_q3EaCk5KqsFNaQ99w4aBSiZk0EbfteEYkTlxEv-O-BLBGkMmSbW6Z66kRHsqn0EVysHeG_w.dQ5mxalHSSfSJEEDZZb40A.z89PuW_vWXFMoJ_XPhIC5KqXUtzdFoB6O4suHDaLrvuKj1H3kNpVhlfwiqAYVqtAPuYm5Ldk7DRdif9ceprogVLLHyRjuut6lldh7y2kWz0PE3zuHam08ZciYa67nHvA.YFxl9NdIy3vcf3N50NsWQg"
+        },
+    
+        "record": {
+            "TXT": "_vercel"
```

Comment:
TXT record alone doesn't do anything.

Reply:
vc-domain-verify=i.is-a.dev,bba748467aca88b3c023 mb i forgot
Register I.is-a.dev with TXT record pointing to vc-domain-verify=i.is-a.dev,bba748467aca88b3c023